docs/source/cli.rst

```diff
@@ -20,7 +25,12 @@ Discovery
 *********
 
 The tool can automatically discover supported devices using a broadcast-based discovery protocol.
-This works by sending an UDP datagram on port 9999 to the broadcast address (defaulting to ``255.255.255.255``).
+This works by sending an UDP datagram on ports 9999 and 20002 to the broadcast address (defaulting to ``255.255.255.255``).
+
+Newer devices that respond on port 20002 will most likely require TP-Link cloud credentials to be passed or they will report as having failed authentication
```

Comment:
```suggestion
Newer devices that respond on port 20002 will require TP-Link cloud credentials to be passed or they will report as having failed authentication
```

Reply:
Devices that have never connected to the cloud should work without authentication so I've updated this to explain that,